### PR TITLE
[FEATURE] Wait time simple implementation

### PR DESCRIPTION
**Pull request by GuilhermeRameh**
Solves #638 
Only changed MINIMUM_WAIT_TIME to 10, as sugested.

### CODE REVIEW

**Comment by FrancescoVassalli** [comment]
This is a bit fast. I worry about people's accounts being flagged by LinkedIn. I would have this be a command line arg. Also if you take a look at [this PR](https://github.com/feder-cr/Auto_Jobs_Applier_AIHawk/pull/662) you can maybe see how one might want to change the medium and long sleep distributions to make ti go faster or slower.

File: app_config.py [hunk 1]
```diff
@@ -12,4 +12,4 @@
 LOG_TO_FILE = True
 LOG_TO_CONSOLE = True
 
-MINIMUM_WAIT_TIME = 60
+MINIMUM_WAIT_TIME = 10
```
Comment (surapuramakhil) [inline app_config.py:15]: can you add unit to varaible

**Review by surapuramakhil — CHANGES_REQUESTED** [review]
can you add unit to variable

File: app_config.py (hunk 1 above)
Comment (GuilhermeRameh) [reply app_config.py:15]: Do you mean as a comment beside the variable identifying its unit, like "# in seconds"?

Comment (surapuramakhil) [reply app_config.py:15]: yes

**Pull request closed without merging**